praw/models/reddit/emoji.py

```diff
@@ -40,11 +40,11 @@ def __eq__(self, other: Union[str, _Emoji]) -> bool:
         """Return whether the other instance equals the current."""
         if isinstance(other, str):
             return other == str(self)
-        return (
-            isinstance(other, self.__class__)
-            and str(self) == str(other)
-            and other.subreddit == self.subreddit
-        )
+        if isinstance(other, self.__class__):
```

Comment:
This change makes sense, but the default should actually be to call `super` instead of `NotImplemented`

Reply:
Makes sense.